src/libs/migrations/PersonalDetailsByAccountID.js

```diff
@@ -225,6 +246,47 @@ export default function () {
                 }
             });
 
+            // For the reports migration, we don't need to look up emails from accountIDs. Instead,
+            // we will just look for old email data and automatically remove it if it exists. The reason for
+            // this is that we already stopped sending email based data in reports, and from everywhere else
+            // in the app by this point (since this is the last data we migrated).
+            _.each(oldReports, (report, onyxKey) => {
+                // Delete report key if it's empty
+                if (_.isEmpty(report)) {
+                    onyxData[onyxKey] = null;
+                    return;
+                }
+
+                let reportWasModified = false;
+                if (lodashHas(report, ['ownerEmail'])) {
+                    reportWasModified = true;
+                    Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing ownerEmail from report ${report.reportID}`);
+                    delete report.ownerEmail;
```

Comment:
To fix the lint define a new const: `const newReport = report;` And use that new const i.e. `delete newReport.ownerEmail;`